services-directions/src/test/java/com/mapbox/api/directions/v5/MapboxDirectionsTest.java

```diff
@@ -517,13 +641,38 @@ public void testRadiusWithUnlimitedDistance() throws IOException {
       .addWaypoint(coordinates.get(1))
       .destination(coordinates.get(2))
       .baseUrl("https://foobar.com")
-      .radiuses(100, Double.POSITIVE_INFINITY, 100)
+      .radiuses(100d, Double.POSITIVE_INFINITY, 100d)
       .build();
 
     assertEquals("100;unlimited;100",
       client.cloneCall().request().url().queryParameter("radiuses"));
   }
 
+  @Test
+  public void testRadiusesListWithUnlimitedDistance() throws IOException {
```

Comment:
We should also remove `test` prefix from tests names, it's implicit with the usage of `@Test` annotation.